ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -131,6 +138,7 @@
 
   private static final int Crc_Size = 8;
   private static final short UserMetadata_Version_V1 = 1;
+  public static final String HTTP_DATE_FORMAT = "EEE, dd MMM yyyy HH:mm:ss zzz";
```

Comment:
If not, we can make changes in `NettyRequest` to export expected `Date` objects as `Date` and not `String`.